indico/modules/rb_new/client/js/modules/admin/reducers.js

```diff
@@ -0,0 +1,49 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {combineReducers} from 'redux';
+import {requestReducer} from 'indico/utils/redux';
+import camelizeKeys from 'indico/utils/camelize';
+import {filterReducerFactory} from '../../common/filters';
+import * as adminActions from './actions';
+
+
+export const initialFilterStateFactory = () => ({
+    showInactive: true,
```

Comment:
default to false? usually you don't care about those (and when showing those maybe we don't want to show active rooms?)

Reply:
Subject to change. The filters aren't available at the moment.